src/SprykerSdk/Integrator/Common/AbstractFacade.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Integrator\Common;
+
+abstract class AbstractFacade
```

Comment:
do we need it this way? Mb an interface should suffice ?

Reply:
As to me it's better to keep `AbstractFacade` which is extended by our facades (IntegratorFacade, ClassBuilderFacade etc) for any future common functionality and at the same time we already have an IntegratorFacadeInterface which already has `runInstallation()` method

Reply:
but for now it's useless. Facades in this module is not a public api so we can addit in the future easily